.travis.yml

```diff
@@ -5,7 +5,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - bundle install
 env:
-  - DISPLAY=':99.0' RAILS_VERSION='3.2.0'
-  - DISPLAY=':99.0'  RAILS_VERSION='4.1.0'
-  - DISPLAY=':99.0'  RAILS_VERSION='4.2.0'
+  - DISPLAY=':99.0' RAILS_VERSION='4.1.0'
+  - DISPLAY=':99.0' RAILS_VERSION='4.2.0'
+  - DISPLAY=':99.0' RAILS_VERSION='5.0.0'
```

Comment:
<img width="1010" alt="screen shot 2016-08-10 at 10 09 05 am" src="https://cloud.githubusercontent.com/assets/150352/17556752/80b3ff34-5ee2-11e6-8c03-a8bc8b62f9f8.png">

This `RAILS_VERSION` tells travis which rails to test with